src/services/videomania/index.jsx

```
@@ -248,3 +255,205 @@ export const AddVideoMania = ({
 };
 export const EditVideoMania = async () => {};
 export const DeleteVideoMania = async () => {};
+
+export const VideoManiaPlayer = ({
+  video,
```

ESLint

[error] 'video' is missing in props validation

(react/prop-types)